core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -221,6 +223,39 @@ class UserServiceSpec extends AnyFlatSpecLike with TestDriverComponent with Mock
     }
   }
 
+  it should "Successfully to delete a billing project when the google project does not exist" in {
```

Comment:
```suggestion
  it should "Successfully to delete a billing project when the google project does not exist on GCP" in {
```

Just want to be explicit because there are `google-project` Resources that exist in Sam too